android/src/main/kotlin/com/apptentive/apptentive_flutter/ApptentiveFlutterPlugin.kt

```diff
@@ -290,6 +291,22 @@ class ApptentiveFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware
     }
   }
 
+  private fun sendAttachmentText(call: MethodCall, result: Result) {
+    val message: String? = call.argument("message")
```

Comment:
@frankus I added this as it was straight forward and cordova has it. Would you be able to provide iOS support?

Reply:
Added and smoke tested